homeassistant/components/denonavr/media_player.py

```diff
@@ -238,7 +239,8 @@ def __init__(
             name=receiver.name,
         )
         self._attr_sound_mode_list = receiver.sound_mode_list
-
+        self._attr_device_class = MediaPlayerDeviceClass.TV
```

Comment:
You can move this outside of the constructor, just like `_attr_name = None` since its static.

Are all denonavr devices a TV?

Reply:
No none of them are TVs, they do not have a display, they are audio video receivers (NVRs) to connect speakers for a sound system.

Reply:
Normally they are connected to a TV though.

Reply:
We could also set it as RECEIVER. 
both device classes also work with HomeKit, so this is fine

Reply:
RECEIVER sounds much better

Reply:
you can move it out, I'll check the samsungtv one

Reply:
#100712